<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductionDialog</class>
 <widget class="QDialog" name="ProductionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabData">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="5">
        <widget class="QComboBox" name="statusCombobox"/>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="jobcodeLabel">
         <property name="text">
          <string>Work order</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="idText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>Output Code</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="jobcodeText">
         <property name="maxLength">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="employeeCombobox"/>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="statusLabel">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="5">
        <widget class="QLineEdit" name="titleText">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="employeeLabel">
         <property name="text">
          <string>Employee</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="idLabel">
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="5">
        <widget class="QPlainTextEdit" name="descriptionText"/>
       </item>
       <item row="1" column="7">
        <widget class="QLineEdit" name="outputText">
         <property name="maxLength">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QLabel" name="supplierLabel">
         <property name="text">
          <string>Supplier</string>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="QComboBox" name="supplierCombobox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNotes">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="authorNoteModEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textNoteLabel">
           <property name="text">
            <string>Text</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="authorNoteModLabel">
           <property name="text">
            <string>Author Modification</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="authorNoteCreationLabel">
           <property name="text">
            <string>Author Creation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="dateNoteCreLabel">
           <property name="text">
            <string>Date Creation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="authorNoteCreEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="dateNoteModLabel">
           <property name="text">
            <string>Date Modification</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDateTimeEdit" name="dateNoteCreEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDateTimeEdit" name="dateNoteModEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QPlainTextEdit" name="textNoteEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="noteTable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>210</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="productionNoteAddButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="productionNoteEditButton">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="productionNoteDeleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>titleText</tabstop>
  <tabstop>idText</tabstop>
  <tabstop>jobcodeText</tabstop>
  <tabstop>outputText</tabstop>
  <tabstop>employeeCombobox</tabstop>
  <tabstop>statusCombobox</tabstop>
  <tabstop>supplierCombobox</tabstop>
  <tabstop>descriptionText</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>productionNoteAddButton</tabstop>
  <tabstop>authorNoteCreEdit</tabstop>
  <tabstop>dateNoteCreEdit</tabstop>
  <tabstop>authorNoteModEdit</tabstop>
  <tabstop>dateNoteModEdit</tabstop>
  <tabstop>textNoteEdit</tabstop>
  <tabstop>noteTable</tabstop>
  <tabstop>productionNoteEditButton</tabstop>
  <tabstop>productionNoteDeleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
